src/baseline/ast/expr.rs

```diff
@@ -992,10 +1006,13 @@ where
                 Intrinsic::IntMul => self.emit_intrinsic_bin_call(e, dest, intrinsic),
                 Intrinsic::IntDiv => self.emit_intrinsic_bin_call(e, dest, intrinsic),
                 Intrinsic::IntMod => self.emit_intrinsic_bin_call(e, dest, intrinsic),
+                Intrinsic::IntNeg => self.emit_intrinsic_unary_call(e, dest, intrinsic),
+                Intrinsic::IntPlus => self.emit_expr(e.object.as_ref().unwrap(), dest),
```

Comment:
Could you please also invoke `emit_intrinsic_unary_call` here as well instead of handling the new intrinsics directly? Then we only have one implementation for them and the code should be more in line with the other intrinsics.

No need to change existing code - just the new intrinsics.

Reply:
Ok, will do!